spec/support/acceptance/child_process.rb

```diff
@@ -22,7 +22,7 @@ class ChildProcess
     def initialize
       super
 
-      @default_timeout = ENV['CI'] ? 120 : 40
+      @default_timeout = ENV['CI'] ? 480 : 40
```

Comment:
Not a blocker for me; We might special case this for just targeting the older PHP releases 😄